examples/favorites_advanced/bloc_favorites_advanced/lib/feature_puppy/search/blocs/puppy_list_state.dart

```diff
@@ -0,0 +1,31 @@
+part of 'puppy_list_bloc.dart';
+
+enum PuppyListStatus { loading, reloading, success, failure }
+
+@immutable
+class PuppyListState {
+  const PuppyListState({
+    required this.status,
+    this.searchedPuppies,
+  });
+
+  const PuppyListState.reloadInProgress(
```

Comment:
@StanislavPrime, what is the difference between `reloadInProgress` and `extraDetailsLoadSuccess` ?